source/change-streams/tests/change-streams-resume-whitelist.yml

```diff
@@ -1105,3 +1105,71 @@ tests:
           fullDocument:
             x:
               $numberInt: "1"
+  -
+    # CursorNotFound is always considered resumable by drivers even though it does not have the
+    # ResumableChangeStreamError label. This test asserts that drivers correctly implement this special-casing for
+    # <=4.2 servers.
+    description: "change stream resumes after CursorNotFound"
+    minServerVersion: "4.2"
+    maxServerVersion: "4.2.99"
+    failPoint:
```

Comment:
On <=4.2, we use a client-side allow-list to determine resumability, so the server error only needs to have a valid code for the test to pass. This can be done by `failCommand`.

On 4.4+, we determine resumability by the presence of the `ResumableChangeStreamError` label, so we need to ensure that label will be present. The server only adds the label if the error code is considered resumable and the cursor in question is a change stream. We can't use `failCommand` for this because it executes before the server actually fetches the cursor based on its ID in the server, so when the error label adding code doesn't know that the cursor is a change stream and therefore leaves off the label.

However, as discussed in the other comment, this is only relevant for the other error codes. Because we're not using error labels for `CursorNotFound,` the 4.4-specific test has been removed altogether.

Reply:
Got it.  Resolving